llm/test/test_llm.py

```diff
@@ -0,0 +1,22 @@
+# test_llm.py
+
+import pytest
+import requests
+
+
+def test_llm_response():
```

Comment:
**praise:** add [mark it requires running LLM](https://docs.pytest.org/en/latest/how-to/mark.html), custom LLM mark or something like that